hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/prehandle/PreHandleWorkflowImpl.java

```diff
@@ -95,6 +97,10 @@ public class PreHandleWorkflowImpl implements PreHandleWorkflow {
      * Used for registering notice of transactionIDs seen by this node
      */
     private final DeduplicationCache deduplicationCache;
+    /**
+     * Used to pre-handle transactions in parallel.
+     */
+    private final ForkJoinPool preHandlePool;
```

Comment:
`preHandlePool` is not used. Parallel transaction pre-handling needs to be restored.

Reply:
I don't understand what the intention with the prehandle changes were, prior to me taking over the PR.

I'm going to close it, and just make a new PR with whatever changes end up getting made in the release 52 branch.